### DISCUSSION TITLE
Brak kompatybilności GET /sale/product-offers/{offerId} ze starą metodą GET /sale/offers/{offerId}

### DISCUSSION THREAD

**Discussion by javaguru-pl** [allegro/allegro-api#6877] [Suggestions - rejected]
Metoda /sale/offers/{offerId} jest oznaczona jako deprecated i przeznaczona do likwidacji. Na jej miejsce powstała /sale/product-offers/{offerId}. Niestety ta nowa metoda nie zwraca niektórych pól starej metody, m.in. sekcji "promotion" zawierającej informacje o promocjach na aukcji. Brakuje także ceny netto "netPrice" w sekcji "sellingMode". Cenę netto można co prawda obliczyć na podstawie procentu podatku VAT, natomiast brak sekcji z promocjami jest poważnym brakiem. Czy można tę metodę uzupełnić o informacje o promocjach dot. oferty??? (wyróżnienie, pogrubienie, podświetlenie, etc.)
Z góry dziękuję za odpowiedź.

**Comment by MaciejFrackowiak** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany. Do zarządzania promowaniem ofert mamy dedykowane zasoby opisane w [naszym poradniku](https://developer.allegro.pl/tutorials/jak-zarzadzac-ofertami-7GzB2L37ase#jak-zarzadzac-promowaniem).
